README.md

```diff
@@ -1,5 +1,6 @@
 # go-github #
 
+![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/google/go-github?sort=semver)
```

Comment:
What if we make this a link like the other banners below?  We could link it to the releases page, like this:

```
[![GitHub release (latest SemVer)](https://img.shields.io/github/v/release/google/go-github?sort=semver)]
(https://github.com/google/go-github/releases)
```